packages/block-library/src/details/save.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * WordPress dependencies
+ */
+import { InnerBlocks, RichText } from '@wordpress/block-editor';
+
+export default ( { attributes } ) => {
+	return (
+		<details open={ attributes.open }>
```

Comment:
This is currently kind of broken since there's no `open` attribute. I'd suggest adding an `initialOpen` attribute that controls whether the accordion is open on page load or not. This attribute could be set from a toggle in the block inspector.

Reply:
Moving it to the block inspector works well for me, it's still an option but it's very explicit.